src/riak_kv_ts_api.erl

```diff
@@ -46,15 +46,15 @@
 -include("riak_kv_ts.hrl").
 
 %% external API calls enumerated
--type query_api_call() :: query_create_table | query_select | query_describe | query_insert | query_show_tables.
+-type query_api_call() :: create_table | query_select | describe_table | query_insert | show_tables.
```

Comment:
These types need to change because the permission list generated in `riak_kv_app:start/2` calls`riak_kv_ts_api:api_calls/0` to register the permssions.